frame/staking/reward-fn/src/lib.rs

```diff
@@ -59,7 +59,7 @@ pub fn compute_inflation<P: PerThing>(
 	ideal_stake: P,
 	falloff: P,
 ) -> P {
-	if stake < ideal_stake {
+	if stake <= ideal_stake {
```

Comment:
@thiolliere this really should be a trivial optimisation, but tests fail with the change.

Reply:
because test are testing for ideal_stake from 0 to 1, when ideal_stake is 0 we should modify the division below into a checked_div, or do a special case in some way